src/utils/type.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function isAString(something) {
+    return typeof something === 'string';
+}
+
+function isAFunction(something) {
+    return typeof something === 'function';
+}
+
+function exists(something) {
```

Comment:
`isDefined`?

Reply:
Yes. Remove this function and use function 'defined' from simple-js-validator